types/spectest/generate/state_comparison/partialsigmessage_MsgSpecTest/msg inconsistent signed message.json

```diff
@@ -4,43 +4,43 @@
 				{
 						"Message": {
 								"Type": 1,
-								"Slot": 12,
+								"Slot": "7413760",
```

Comment:
@MatheusFranco99 why did this change do you know?

Reply:
Yes. The testing proposer duty for the Deneb version has a duty slot that is higher than the Deneb fork slot.